controllers/articles.controller.js

```diff
@@ -1,6 +1,15 @@
-const { getArticle, checkArticleExists } = require("../models/articles.model")
+const { getArticle, checkArticleExists, getAllArticlesObj } = require("../models/articles.model")
 
- 
+exports.getAllArticles = (req, res, next) => {
+    const query = req.query
+    var size = Object.keys(query).length;
```

Comment:
Please dont declare variables with var, using let/const is always going to be more favourable.